components/licence-header.jsx

```diff
@@ -1,30 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { LicenceSignIn } from './licence-sign-in';
+import { LicenceTitle } from './licence-title';
 
-export function LicenceHeader ({
-	displayName = null,
-	isTrial = false,
-	isB2cPartnershipLicence = false,
-	welcomeText = null
-}) {
+export function LicenceHeader (props) {
+	 const {
+		displayName = null,
+		isTrial = false,
+		isB2cPartnershipLicence = false,
+		welcomeText = null,
+		url
```

Comment:
not default value for the url, so I guess it should be required?
or we need to add default?
```suggestion
		url = '',
```